modules/biobank/php/containercontroller.class.inc

```diff
@@ -0,0 +1,405 @@
+<?php declare(strict_types=1);
+/**
+ * This class features the code that enables access to the containers of the
+ * LORIS Biobank Module.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Controller
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.text GPLv3
+ * @link       http://www.github.com/aces/Loris/
+ */
+namespace LORIS\biobank;
+
+/**
+ * This class features the code that enables access to the containers of the
+ * LORIS Biobank Module.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Controller
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.text GPLv3
+ * @link       http://www.github.com/aces/Loris/
+ */
+class ContainerController
+{
+    /**
+     * Global variable $db
+     *
+     * @access private
+     */
+    private $db;
+
+    /**
+     * Global variable $user
+     *
+     * @access private
+     */
+    private $user;
+
+    /**
+     * Global variable $dao
+     *
+     * @access private
+     */
+    private $dao;
+
+    /**
+     * Class Constructor
+     *
+     * @param Database $db   Database Instance.
+     * @param User     $user User Instance.
+     */
+    function __construct(\Database $db, \User $user)
+    {
+        $this->db   = $db;
```

Comment:
Is there a reason for this to be a class variable instead of just using the singleton internally?

Reply:
Yeah, I don't really think there's a reason to be using singleton whenever a $db is used when it can be set upon instantiation. I think a controller should be tied to a user, db and dao instantiation and shouldn't be constructed without them.